sport/app/football/views/matchStats/matchStatsPage.scala.html

```diff
@@ -3,8 +3,7 @@
 @import implicits.Football._
 @import views.FootballHelpers._
 
-
-@(page: MatchPage, competition: Option[model.Competition])(implicit request: RequestHeader, context: model.ApplicationContext)
+@(page: _root_.football.controllers.MatchPage, competition: Option[model.Competition])(implicit request: RequestHeader, context: model.ApplicationContext)
```

Comment:
I'm not sure I understand why these have to be `_root_`, is there another `MatchPage` it might pick up otherwise?